packages/apollo-server-express/src/__tests__/datasource.test.ts

```diff
@@ -32,10 +33,10 @@ const typeDefs = gql`
 
 const resolvers = {
```

Comment:
Alternatively, this could probably be typed using https://github.com/apollographql/apollo-tooling/blob/master/packages/apollo-graphql/src/schema/resolverMap.ts

Reply:
ah will give that a quick shot